src/app/public/modules/back-to-top/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './back-to-top.component';
```

Comment:
Shouldn't export the component to the public API.

Reply:
Also, to prepare for Angular 9, it would be better to delete this `index.ts` file and put all exports on the `src/app/public/index.ts` file.